tests/acceptance/bootstrap/GraphContext.php

```diff
@@ -2823,6 +2823,54 @@ public function userListsTheActivitiesForResourceOfSpaceUsingTheGraphAPI(string
 		$this->featureContext->setResponse($response);
 	}
 
+	/**
+	 * @When user :user tries to list the activities of folder :folder with share mount-point id using the Graph API
+	 *
+	 * @param string $user
+	 * @param string $folder
+	 *
+	 * @return void
+	 */
+	public function userListsTheActivitiesOfFolderFromSpaceWithShareMountIdPointIdUsingTheGraphApi(string $user, string $folder): void {
```

Comment:
let's name the step defs as close as possible to step name
```suggestion
	public function userTriesToListTheActivitiesOfFolderWithShareMountIdPointIdUsingTheGraphApi(string $user, string $folder): void {
```

Reply:
Sure.